python_ta/transforms/z3_visitor.py

```diff
@@ -30,7 +30,7 @@ def set_function_def_z3_constraints(self, node: nodes.FunctionDef):
         for ann, arg in zip(annotations, arguments):
             if ann is None:
                 continue
-            # TODO: what to do about subscripts ex. Set[int], List[Set[int]], ...
+            # TODO: what to do about subscripts ex. Set[int], list[Set[int]], ...
```

Comment:
It's actually fine to change this, but you should be consistent and also change the `Set`